test/uk/gov/hmrc/cbcr/controllers/CBCIdControllerSpec.scala

```diff
@@ -40,11 +41,9 @@ class CBCIdControllerSpec
 
   val localGen = mock[LocalSubscription]
   val remoteGen = mock[RemoteSubscription]
-  var runMode = mock[RunMode]
-  when(runMode.env) thenReturn "Dev"
 
   implicit val as = app.injector.instanceOf[ActorSystem]
-  val config = app.injector.instanceOf[Configuration]
+  val config = mock[ApplicationConfig]
```

Comment:
With this mock on configuration now in our tests, am wondering if this now masks any exceptions we get due to missing default values in config.

Reply:
Yes, but I wouldn't worry about it since the application is used as-is in acceptance tests, so if there's any problem with missing configs, that would pick it up.

Reply:
Yeah. Was more in light of now we will have to run ATs to verify this due to the mock whereas before a normal unit/intesgration-test build would've caught config errors.

But, I guess as a good practice we run ATs aswell anyway 👍